modules/ipi-install-network-requirements.adoc

```diff
@@ -160,5 +160,5 @@ You can reconfigure the control plane nodes to act as NTP servers on disconnecte
 
 [id='network-requirements-out-of-band_{context}']
 == Port access for the out-of-band management IP address
-
-The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the `baremetal` node during installation, the out-of-band management IP address must be granted access to port `6180`.
+`
```

Comment:
You could remove this back-tick so that this is just a blank line. At the moment it is showing up in the rendered build as follows:

> ` The out-of-band management...